app/models/ordergroup.rb

```diff
@@ -36,6 +36,7 @@ def self.include_transaction_class_sum
     select(columns.join(', '))
       .joins('LEFT JOIN financial_transactions ON groups.id = financial_transactions.ordergroup_id')
       .joins('LEFT JOIN financial_transaction_types ON financial_transaction_types.id = financial_transactions.financial_transaction_type_id')
+      .where(financial_transactions: {reverts: nil})
```

Comment:
I don't quite get why this added where-clause is necessary.

Reply:
some code from an older revision 😨